04_setup_ironic.sh

```diff
@@ -134,6 +134,30 @@ if [ ! -z "${MIRROR_IMAGES}" ]; then
     IRONIC_RELEASE_IMAGE=$(image_for ironic | cut -d '@' -f2)
     LOCAL_REGISTRY_PREFIX="${LOCAL_REGISTRY_DNS_NAME}:${LOCAL_REGISTRY_PORT}/localimages/local-release-image"
     IRONIC_LOCAL_IMAGE=${IRONIC_LOCAL_IMAGE:-"${LOCAL_REGISTRY_PREFIX}@${IRONIC_RELEASE_IMAGE}"}
+
+    if [ ! -z "${MIRROR_OLM}" ]; then
```

Comment:
The CI job is failing because this is undefined by default:

```
./04_setup_ironic.sh: line 138: MIRROR_OLM: unbound variable
```

So we need to either default it in `common.sh` or here e.g `${MIRROR_OLM:-}`

Reply:
Ah, oops. I didn't realize this script was set -u. I've fixed that and the double-negation ! -z to just -n.